locales/en/allocations.json

```diff
@@ -24,5 +24,36 @@
   },
   "allocation_criteria": {
     "page_title": "Allocation criteria"
+  },
+  "view": {
+    "page_title": "View allocation",
+    "heading": "{{count}} person from {{from_location}} to {{to_location}}, {{date}}",
+    "heading_plural": "{{count}} people from {{from_location}} to {{to_location}}, {{date}}",
+    "criteria_for_allocation": "Criteria for allocation",
+    "created_at": "Created {{datetime}}",
+    "fields": {
+      "estate_type": "Estate type",
+      "category": "Category",
+      "complex_cases": "Complex cases for prisons to agree",
+      "none_provided": "None provided",
+      "other_criteria": "Other criteria",
+      "sentence_length": {
+        "label": "Time left to serve"
+      }
+    },
+    "complete_in_full": "This allocation should be filled in full - consider prisoners that are the next best for this allocation",
+    "people": {
+      "slots_filled": "{{slots_filled}} of {{total_slots}} {{person_or_people}} allocated",
+      "heading": "People for allocation",
+      "empty_slot": "People to be allocated"
+    },
+    "cancel_allocation": "Cancel allocation",
+    "allocation_details": "Allocation details",
+    "sidebar": {
+      "number_of_prisoners": "Number of people",
+      "move_from": "Move from",
+      "move_to": "Move to",
+      "date": "Date"
```

Comment:
Same here. This label already exists within moves. We should re-use where possible for consistency.

Reply:
See above. It's not clear to me if our policy is to reuse or to err on the side of caution. Personally I'd rather not recycle as it could lead to unintended consequences down the line, if for example the copy is radically changed for one journey in a way that doesn't fit the other.

Reply:
Again, it depends on the context. In the case here where we want to create unity between how we refer to a from and to location I think it makes sense to re-use.

And also in the case of labels. Unless they are drastically different we should use the same labels when capturing the data and playing it back so a user is aware what it refers to.

Often it's a judgement call though, there isn't a one-size-fits-all policy for this.

Reply:
The lack of policy in general, is very disempowering. It seems to me that whatever choice I make, the other way will be argued for, which leads to non-uniformity and general waste of time / energies. I think that a blanket policy may leave some edge cases out, but should nonetheless be decided for, and exceptions evaluated if they are such, e.g. exceptions.

Reply:
The default policy has always been to re-use code and translation to ensure consistency. Especially when we are describing related resources. 

When we are talking about a different resource, it is better to use a separate translation key.